### PR TITLE
Fix issue in native parser of cli_parse plugin

### PR DESCRIPTION
**Pull request by ashwini-mhatre**
##### SUMMARY

fixes : https://github.com/ansible-collections/ansible.netcommon/issues/347
Depends-on: https://github.com/ansible-collections/ansible.netcommon/pull/357

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

cli_parse 

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by ashwini-mhatre** [comment]
recheck

**Comment by ashwini-mhatre** [comment]
recheck

File: tests/integration/targets/cli_parse/tasks/centos_native.yaml [hunk 1]
```diff
@@ -5,3 +5,17 @@
     parser:
       name: ansible.netcommon.native
     set_fact: myfact
+
+- name: "{{ parser }} Run command which output nothing and parse with native"
+  ansible.utils.cli_parse:
+    command: "touch test_output"
+    parser:
+      name: ansible.netcommon.native
+    ignore_errors: true
```
Comment (Qalthos) [inline tests/integration/targets/cli_parse/tasks/centos_native.yaml:14]: Why are we specifying ignore_errors here?

**Review by Qalthos — CHANGES_REQUESTED** [review]

File: tests/integration/targets/cli_parse/tasks/centos_native.yaml (hunk 1 above)
Comment (ashwini-mhatre) [reply tests/integration/targets/cli_parse/tasks/centos_native.yaml:14]: Ack

File: changelogs/fragments/bugfix_cli_parse_native_parser.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - Fix issue in native parser of Cli_parse plugin.
```
[the remark this answers is not in the document]
Comment (ashwini-mhatre) [reply changelogs/fragments/bugfix_cli_parse_native_parser.yaml:3]: ack

**Comment by ashwini-mhatre** [comment]
recheck

**Comment by ashwini-mhatre** [comment]
recheck

**Comment by ashwini-mhatre** [comment]
recheck

**Comment by ashwini-mhatre** [comment]
recheck

**Review by Qalthos — APPROVED** [review]

**Comment by Qalthos** [comment]
recheck

**Comment by Qalthos** [comment]
recheck

**Comment by Qalthos** [comment]
recheck

**Pull request merged**